source/db/en-projects.json

```diff
@@ -3918,6 +3918,117 @@
   },
   {
     "development_stage": "released",
+    "description": "A safe home for all your data",
+    "license_url": "https://github.com/nextcloud/server/blob/master/COPYING-README",
+    "logo": "nextcloud.png",
+    "notes": "Nextcloud provides a file sync & share solution that respects your privacy and is fully open source. Sync clients are available for all major platforms. Nextcloud comes with a rich ecosystem of apps including Contacts, Calendar, Video calls, Mail, Newsreader and much more.",
+    "privacy_url": "",
+    "source_url": "https://github.com/nextcloud",
+    "name": "Nextcloud",
+    "tos_url": "",
+    "url": "http://nextcloud.com",
+    "wikipedia_url": "https://en.wikipedia.org/wiki/Nextcloud",
+    "protocols": [
+      "CalDAV",
+      "CardDAV",
+      "IMAP",
```

Comment:
We don't currently have an `IMAP` protocol. Possibly we should, but in the meantime if you can get rid of this line that'd be great :)